extendeddatasquare_test.go

```diff
@@ -26,6 +27,31 @@ func TestComputeExtendedDataSquare(t *testing.T) {
 	}
 }
 
+func TestMarshalJSON(t *testing.T) {
+	codec := NewRSGF8Codec()
+	result, err := ComputeExtendedDataSquare([][]byte{
+		{1}, {2},
+		{3}, {4},
+	}, codec, NewDefaultTree)
+	if err != nil {
+		panic(err)
```

Comment:
y panic and not just failnow? you want stacktrace? ( in general in tests for rsmt2d )

Reply:
Probably for the stacktrace - this is how it's done in the other tests